### ISSUE TITLE
Annotate type of setUserProperties dictionary to support only string values

### ISSUE THREAD

**Issue by marcknock** [freshdesk/freshchat-ios#84]
You only support string as values, please annotate this in your API to avoid us the (nice) warning at runtime.

```
-(void)setUserProperties:(NSDictionary*)props
```

```
-(void)setUserProperties:(NSDictionary<NSString *, NSString*> *)props;
```

**Comment by harish-kumarc** [comment]
Thanks for the feedback @marcknock  we will update that.

**Comment by harish-kumarc** [comment]
HI @marcknock we have fixed this with release v3.3.3(https://github.com/freshdesk/freshchat-ios/releases/tag/v3.3.3). Please update to latest SDK.
